docs/about/how-to-create-and-submit-PR.md

```diff
@@ -0,0 +1,110 @@
+
+![Image showing the labels side panel](/img/create_pr/labels.png)
+
+## Submitting a PR :tada:
+
+**Step 9:**  Proofread and do a once-over to ensure all the necessary details are mentioned in your PR. When you feel it is good to go, click that green “Create pull request” button!
+
+You should now see your PR in the [Pull requests page](https://github.com/transcom/mymove/pulls).
+
+  
+**Step 10:** Go back to your Jira board. Move your ticket from the “Started” column to the “In review” column.
+
+## Merging your PR :100:
+
+**Step 11**:  Once you’ve received the necessary approvals for your PR and all the checks have passed, it is time to merge it into main.
+
+**FIRST APPROACH**
+
+a)  Scroll down in your PR page, towards the bottom you’ll see a list of checks that get run. If you’re making changes in the Frontend, you’ll usually get Happo diffs and a failing test for this as a result. Make sure a designer also approves your Happo diffs prior to merging your PR.
+![Image showing happo diffs](/img/create_pr/happo_diffs.png)
+
+**NOTE:** If any of your CircleCI tests fail, click the “details” link to get more info.
+
+  
+
+1.  Depending on whether your ticket is frontend or backend, you should run `make_client test` or `make_server test` locally respectively, to make sure all tests are passing before you commit your changes & submit a PR.<br/>
+-If client test or server test fails, that means either a) the code changes you made affected another thing and it caused the failing test or b) you need to make revisions and adapt the tests to account for the changes you’ve made.<br/>
+-You can also run Cypress locally with `make e2e_test` to reproduce the issue.
+
+2.  Cypress integration tests can sometimes be flaky as denoted by the ‘flaky’ CircleCI labels and can fail sporadically as well. These tests you can try re-running if it fails.
+
+b)  Make sure to regularly pull from main to keep your branch updated with the latest before merging. Otherwise, you’ll get a warning that states ‘branch has conflicts…’
+
+![Image showing merge conflicts](/img/create_pr/merge_conflicts.png)
+
+c) Once you see 3 green checks for a) changes approved, b) all checks passed, c) branch has no conflicts, you’ll see the bottom green button “Squash and merge” is enabled. Go ahead and **CLICK THIS BUTTON!**
+
+![Image showing merge button](/img/create_pr/merge.png)
+
+**SECOND APPROACH**
+
+Another way of merging your changes is using Mergify.
+
+**Pros of using Mergify**
+-   Don't have to manually merge main and can ensure that your code passes the tests before merging. Especially if you forget to pull the latest from main, you may come across tests breaking. Mergify helps you here.
+
+**Con of using Mergify**
+-   Can take a while, especially if there are multiple merges happening at the same time. Mergify has a queue of jobs that it tries to merge in order so you may be blocked by other merges. Nonetheless, it is very seamless and simple.
```

Comment:
**suggestion:** I've also found Mergify to get a bit too merge happy when getting an approval, i.e. it merges things that only have design review when the PR should really have both design and code review. Could we mention this as well?

Reply:
@YanZ777 done. I have added this info in. I also made a note that the person should add the "ready-to-merge" label only after they know they've gotten all the necessary approvals and reviews